product_docs/docs/migration_toolkit/55/08_mtk_command_options.mdx

```diff
@@ -9,6 +9,9 @@ Append migration options when you run Migration Toolkit to conveniently control
 
 `$ ./runMTK.sh -allSchemas`
 
+!!! Note
+    The `-allschemas` parameter is supported only for the Oracle, EDB Postgres Advanced Server, and PostgreSQL source database. It is not supported for Sybase, MS SQL Server, and MySQL source databases.
```

Comment:
Should databases be plural in the first sentence?

Reply:
Yes.